pkg/morph/client/container/load.go

```diff
@@ -53,34 +54,105 @@ type EstimationID []byte
 
 // ListLoadEstimationsByEpoch returns a list of container load estimations for to the specified epoch.
 // The list is composed through Container contract call.
-func (c *Client) ListLoadEstimationsByEpoch(epoch uint64) ([]EstimationID, error) {
+func (c *Client) ListLoadEstimationsByEpoch(epoch uint64) ([]*Estimations, error) {
 	invokePrm := client.TestInvokePrm{}
 	invokePrm.SetMethod(listSizesMethod)
 	invokePrm.SetArgs(epoch)
 
-	prms, err := c.client.TestInvoke(invokePrm)
+	kvs, err := c.client.TestInvokeIterator(invokePrm)
 	if err != nil {
 		return nil, fmt.Errorf("could not perform test invocation (%s): %w", listSizesMethod, err)
-	} else if ln := len(prms); ln != 1 {
-		return nil, fmt.Errorf("unexpected stack item count (%s): %d", listSizesMethod, ln)
 	}
 
-	prms, err = client.ArrayFromStackItem(prms[0])
-	if err != nil {
-		return nil, fmt.Errorf("could not get stack item array from stack item (%s): %w", listSizesMethod, err)
-	}
+	resMap := make(map[cid.ID]*Estimations)
+	for i := range kvs {
```

Comment:
Can't we use an `unwrap.Map` for it? Is it a map returned from the contract?

Reply:
Nope, it's not.